site/README.md

```diff
@@ -1,70 +1,106 @@
 # Helsinki Design System documentation site
 
-Built with [docz](https://www.docz.site/).
+This is the public documentation site of Helsinki Design System ([hds.hel.fi](https://hds.hel.fi/)). It is built with [docz](https://www.docz.site/) which is powered by [Gatsby](https://www.gatsbyjs.com/). Documentation site includes:
+
+- Getting started section for designers and developers
+- Guidelines for accessibility, grid, localisation and more
+- Guidelines for visual assets such as icons and Helsinki logo
+- Design token documentation
+- Component documentation
+- About section with news, release notes and road map
+- Resources section
+- Contribution guidelines
 
 ## Getting started
 
-### Run the dev env
+When making changes to the documentation website, recommended way to get it running locally. This way you can review and see your changes in real time like they would show up in production. 
```

Comment:
"recommended way to get it running locally" lauseeseen on jääny joku vähän oudosti

Reply:
Korjattu.